usersprompt.js

```diff
@@ -0,0 +1,32 @@
+var readline = require('readline');
+
+var rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+function logAgeGroup(number) {
+    if (number < 18) {
+        console.log('User is a child');
+    }
+    else if (number < 36) {
+        console.log('User is a youth')
+    }
+    else {
+        console.log('User is an adult')
+    }
+}
+
+rl.question('What is your Year of Birth?: ', function (x) {
+    var year = parseInt(x)
+    if (year >= 1900) {
```

Comment:
Lets change the order of some lines like this

1. Put `logAgeGroup(age)` below `var age = 2019 - year`
2. Only call `r.close()` at the end of the function and remove it from the `else` block